packages/block-editor/src/components/block-title/index.js

```diff
@@ -14,12 +14,23 @@ import { getBlockType } from '@wordpress/blocks';
  * <BlockTitle clientId="afd1cb17-2c08-4e7a-91be-007ba7ddc3a1" />
  * ```
  *
- * @param {Object}  props
- * @param {?string} props.name Block name.
+ * @param {Object} props
+ * @param {string} props.clientId Client ID of block.
  *
  * @return {?string} Block title.
  */
-export function BlockTitle( { name } ) {
+function BlockTitle( { clientId } ) {
```

Comment:
You can do `export default function BlockTitle(...` here rather than exporting it later.

Reply:
I can't confirm... But it looks like it was previously exported on its own for unit testing purposes. It looks like it might be safe to do a `export default` now

Reply:
I guess Zeb is suggesting the export in the same line as the definition of the function, I personally prefer separate for default exports. I find it clearer but you know these are just style opinions, so not important.

Reply:
Yeah, not really that important. :+1: